src/wp-admin/includes/plugin.php

```diff
@@ -2473,3 +2473,74 @@ function paused_plugins_notice() {
 		__( 'Go to the Plugins screen' )
 	);
 }
+
+/**
+ * Renders an admin notice in case some plugins deactivated.
+ *
+ * Displays an admin notice in case a plugin has been deactivated during an
+ * upgrade due to incompatibility with the current version of WordPress.
+ *
+ * @since 5.8.0
+ * @access private
+ *
+ * @global string $pagenow
+ */
+function wp_deactivated_plugins_notice() {
+	if ( 'plugins.php' === $GLOBALS['pagenow'] ) {
+		return;
+	}
+
+	if ( ! current_user_can( 'activate_plugins' ) ) {
+		return;
+	}
+
+	$deactivated_plugins = get_option( 'wp_deactivated_plugins' );
+
+	if ( false === $deactivated_plugins ) {
+		// Option not in database, add an empty array to avoid extra DB queries on subsequent loads.
+		update_option( 'wp_deactivated_plugins', array() );
+		return;
+	}
```

Comment:
May be better to add an `upgrade_580()` to https://github.com/WordPress/wordpress-develop/blob/master/src/wp-admin/includes/upgrade.php#L689. Something like:

```
function upgrade_580() {
    // Do add_option() to not overwrite existing option.
    add_option( 'wp_deactivated_plugins', array() );
}
```